RSSReader-Maruyama/AppDelegate.swift

```diff
@@ -7,17 +7,39 @@
 //
 
 import UIKit
+import FBSDKCoreKit
+import GoogleSignIn
+import Keys
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        
+        ApplicationDelegate.shared.application( application, didFinishLaunchingWithOptions: launchOptions )
+        
+        let keys = RSSReaderMaruyamaKeys()
+//        Google SignIn：クライアントIDの設定
+        GIDSignIn.sharedInstance()?.clientID = keys.gIDSignInClientID
+//        FacebookAppIDの設定
+        Settings.appID = keys.facebookAppID
+        
         return true
     }
 
+    func application( _ app:UIApplication, open url:URL, options: [UIApplication.OpenURLOptionsKey :Any] = [:] ) -> Bool {
+        ApplicationDelegate.shared.application( app, open: url,
+                                                sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
+                                                annotation: options[UIApplication.OpenURLOptionsKey.annotation] )
```

Comment:
```suggestion
        ApplicationDelegate.shared.application(app,
                                               open: url,
                                               sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
                                               annotation: options[UIApplication.OpenURLOptionsKey.annotation])
```
メソッド呼び出し時に改行を入れる場合は引数毎に改行するようにしてください。